package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-config-tc",
-  "version": "12.0.0",
+  "version": "12.1.0",
```

Comment:
This should probably be `13.0.0` since we are adding a breaking rule, but we can do that next time 🤓